### ISSUE TITLE
Add to HACS

### ISSUE THREAD

**Issue by tylwright** [TBens/lovelace-ui-minimalist#6]
Seems like it'd be beneficial to add this to [HACS](https://hacs.xyz).  Thoughts?

**Comment by ghost** [comment]
Yes PLEASE. I'm incredibly clueless as to how to install it manually.

**Comment by joshtbernstein** [comment]
I think it currently requires to much customization to implement with HACS. There's a lot of things specific to @TBens own HA config in the code.
